packages/map-template/README.md

```diff
@@ -1,88 +1,114 @@
+In your script:
 
-### Adding Google Maps API Keys or Mapbox Access Tokens
+```javascript
+import MapsIndoorsMap from '@mapspeople/map-template/dist/mapsindoors-webcomponent.es.js';
+window.customElements.define('mapsindoors-map', MapsIndoorsMap);
+```
 
-Rename `.env.example`, to `.env` and add either of the keys (or both) to that file, and the maps will load properly.
+In your styles make sure to give it a size. For example:
 
-### React Components
+```css
+mapsindoors-map {
+    display: block;
+    width: 100vw;
+    height: 100vh;
+}
+```
 
-We follow [the React convention](https://reactjs.org/docs/faq-structure.html#avoid-too-much-nesting) with one level of depth in the Components' folder. We still import React components into each other (even in multiple levels), but the folder structure is flat.
+Make sure the MapsIndoors JavaScript SDK is loaded by having this somewhere in your HTML:
 
-### Web Components
+```html
+<script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.21.5/mapsindoors-4.21.5.js.gz"></script>
+```
 
-We use our own Web Components ([Custom Elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)) from `packages/components` inside of the Map Template. To illustrate the relation between the React app, its React Components and our Web Components, here's a drawing of how this works in the Route Instructions part of the app:
+Use the web component in your HTML:
 
-```
-// public/index.html
-
-- App.js
- - MapsIndoorsMap
-  - BottomSheet
-   - Directions
-    - RouteInstructions
-     - mi-route-instructions-step
-      - mi-icon
-      - mi-time
-      - mi-distance
+```html
+<mapsindoors-map></mapsindoors-map>
 ```
 
-The `mi-route-instructions-step` Web Component uses `mi-route-instructions-maneuver` to render sub steps, `mi-icon` for icons, and `mi-time` and `mi-distance` to show estimated time and distance.
-
-We use `mi-route-instructions-step` in the React Component `RouteInstructions` to show, well, route instructions. This is used inside of an "umbrella" component for all directions (including searching for where to go) called `Directions`.
+Add attributes to the web component as needed (see supported properties below).
+
+## Using just the browser
+
+```html
+<script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.21.5/mapsindoors-4.21.5.js.gz"></script>
+<script type="module">
+    import MapsindoorsMap from 'https://www.unpkg.com/@mapspeople/map-template/dist/mapsindoors-webcomponent.es.js';
+    window.customElements.define('mapsindoors-map', MapsIndoorsMap)
+</script>
+<style>
+      body {
+            margin: 0;
+      }
+      mapsindoors-map {
+            display: block;
+            width: 100vw;
+            height: 100vh;
+      }
+</style>
+
+<mapsindoors-map></mapsindoors-map>
+```
```

Comment:
I think it could be great to have this as a complete example, like so:

```suggestion
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>My Map</title>
    <script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.21.5/mapsindoors-4.21.5.js.gz"></script>
    <script type="module">
        import MapsindoorsMap from 'https://www.unpkg.com/@mapspeople/map-template/dist/mapsindoors-webcomponent.es.js';
        window.customElements.define('mapsindoors-map', MapsIndoorsMap)
    </script>
    <style>
        body {
                margin: 0;
        }
        mapsindoors-map {
                display: block;
                width: 100vw;
                height: 100vh;
        }
    </style>
</head>
<body>
    <mapsindoors-map></mapsindoors-map>
</body>
</html>
```
```